### PR TITLE
Polish minor cn words

### PR DESCRIPTION
**Pull request by dreamerlin**

### CODE REVIEW

File: configs/recognition/tsm/tsm_mobilenetv2_dense_1x1x8_100e_kinetics400_rgb.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 _base_ = [
     '../../_base_/models/tsm_mobilenet_v2.py',
-    '../../_base_/schedules/sgd_tsm_mobilenet_v2_50e.py',
+    '../../_base_/schedules/sgd_tsm_mobilenet_v2_100e.py',
```
Comment (innerlee) [inline configs/recognition/tsm/tsm_mobilenetv2_dense_1x1x8_100e_kinetics400_rgb.py:3]: Is this a bug fix?

File: configs/recognition/tsm/tsm_mobilenetv2_dense_1x1x8_100e_kinetics400_rgb.py [hunk 2]
```diff
@@ -48,7 +48,7 @@
 ]
 test_pipeline = [
     dict(
-        type='SampleFrames',
+        type='DenseSampleFrames',
```
Comment (innerlee) [inline configs/recognition/tsm/tsm_mobilenetv2_dense_1x1x8_100e_kinetics400_rgb.py:51]: Will this change affect testing results in README?

File: configs/localization/bmn/README_zh-CN.md [hunk 3]
```diff
@@ -37,7 +37,7 @@
 | |mmaction_clip |2|75.35|67.38|43.08|32.19|10.73|31.15|5420|3.27|[ckpt](https://download.openmmlab.com/mmaction/localization/bmn/bmn_400x100_2x8_9e_mmaction_clip/bmn_400x100_2x8_9e_mmaction_clip_20200809-10d803ce.pth)| [log](https://download.openmmlab.com/mmaction/localization/bmn/bmn_400x100_2x8_9e_mmaction_clip/bmn_400x100_2x8_9e_mmaction_clip_20200809.log) | [json](https://download.openmmlab.com/mmaction/localization/bmn/bmn_400x100_2x8_9e_mmaction_clip/bmn_400x100_2x8_9e_mmaction_clip_20200809.json) |
 | [BMN-official](https://github.com/JJBOY/BMN-Boundary-Matching-Network) (for reference)* |cuhk_mean_100 |-|75.27|67.49|42.22|30.98|9.22|30.00|-|-|-| - | - |
 
-- Notes:
+- 注意：
```
Comment (innerlee) [inline configs/localization/bmn/README_zh-CN.md:40]: 注意 ->  注

怎样

File: configs/recognition/tsm/tsm_mobilenetv2_dense_1x1x8_100e_kinetics400_rgb.py (hunk 2 above)
Comment (dreamerlin) [reply configs/recognition/tsm/tsm_mobilenetv2_dense_1x1x8_100e_kinetics400_rgb.py:51]: No, it is a typo and doesn't affect the test results

File: configs/localization/bmn/README_zh-CN.md (hunk 3 above)
Comment (dreamerlin) [reply configs/localization/bmn/README_zh-CN.md:40]: They are the same

File: configs/recognition/tsm/tsm_mobilenetv2_dense_1x1x8_100e_kinetics400_rgb.py (hunk 1 above)
Comment (dreamerlin) [reply configs/recognition/tsm/tsm_mobilenetv2_dense_1x1x8_100e_kinetics400_rgb.py:3]: typo

File: configs/localization/bmn/README_zh-CN.md (hunk 3 above)
Comment (innerlee) [reply configs/localization/bmn/README_zh-CN.md:40]: 注意更多是 pay attention to, be careful
注更多是 note, explain, comment on

我希望我们的文档 less aggressive and startling, more polite and gentle

Comment (dreamerlin) [reply configs/localization/bmn/README_zh-CN.md]: modified

**Review by innerlee — APPROVED** [review]

**Pull request merged**